styles.css

```diff
@@ -0,0 +1,160 @@
+* {
+  font-family: poppins, sans-serif;
+  color: #172b4d;
+}
+
+.head {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  margin: 15px;
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+}
+
+.h {
+  height: 150px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-top: 114px;
+  margin-left: 10px;
+}
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to use replicate [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) on your text elements by using the same ```Font-size``` and ```color``` 👍 👍 

- [ ] **[Optional]** It would be better to improve the spacing on your webpage to match that of Figma design

**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/205434699-942d5827-8e01-4492-a1d6-e8b619b4fe3f.png) | ![image](https://user-images.githubusercontent.com/102757126/205434766-0f63ad6e-f9f7-4afb-9396-f085c7df2ec7.png)